src/specs/models/PipelineResponse.v1.yaml

```diff
@@ -0,0 +1,43 @@
+title: Pipeline response
+type: object
+description: This is the format the pipeline clients communicate the results of the steps they completed back to the API. The API is then responsible for transforming this data and returning it back to the user.
+properties:
+  output:
+    type: object
+    required:
+      - bucket
+      - key
+    properties:
+      bucket:
+        type: string
+        description: The S3 bucket where the response body is located.
+      key:
+        type: string
+        description: The key within the S3 bucket where the result is located.
+  input:
+    type: object
+    properties:
+      experimentId:
+        type: string
+        description: The experiment ID for which the step was completed.
+      taskName:
```

Comment:
it is most likely out of scope, but curious anyway: what will happen if from the UI the user assembled a pipeline with two tasks having duplicate names?